docs/multicast-guide.md

```diff
@@ -0,0 +1,122 @@
+# Multicast User Guide
+
+Antrea supports Multicast traffic in the following scenarios:
+
+1. Pods to Pods, any Pod that has joined the multicast group should receive the traffic from the Pod senders send multicast traffic to that group.
+2. Pods to External, external hosts connected to the Node will receive the multicast traffic from that local Pods.
+3. External to Pod, multicast traffic sent by external hosts connected to the Nodes will be received if some local Pods have joined the multicast group.
+4. Mixture of cases from 1,2,3.
+
+## Table of Contents
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Multicast Networkpolicy](#multicast-networkpolicy)
+- [Debugging and Collecting Multicast Statistics](#debugging-and-collecting-multicast-statistics)
+  - [Pods join multicast group information](#pods-join-multicast-group-information)
+  - [Inbound and Outbound traffic statistics of multicast](#inbound-and-outbound-traffic-statistics-of-multicast)
+  - [Multicast Networkpolicy Statistics](#multicast-networkpolicy-statistics)
+- [Use example](#use-example)
+- [Limitations](#limitations)
+  - [Traffic in Local Network Control Block](#traffic-in-local-network-control-block)
+  - [Linux kernel](#linux-kernel)
+  - [Encap mode](#encap-mode)
+<!-- /toc -->
+
+## Prerequisites
+
+`Multicast` is introduced in v1.5 as an alpha feature. The feature gate
+`Multicast` must be enabled in the `antrea-controller` and `antrea-agent`
+configuration.
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable multicast traffic.
+      Multicast: true
+  antrea-agent.conf: |
+    # Enable multicast traffic.
+      Multicast: true
+    multicast:
+    # The names of the interfaces on Nodes that are used to forward multicast traffic.
+    # Defaults to transport interface if not set.
+      multicastInterfaces: 
+
+    # The interval at which the antrea-agent sends IGMP queries to Pods.
+    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+      igmpQueryInterval: "125s"
+```
+## Multicast Networkpolicy
```

Comment:
```suggestion
## Multicast NetworkPolicy
```
Please check other `Networkpolicy`, it should be `NetworkPolicy`

Reply:
updated